src/helper/helper.js

```diff
@@ -310,20 +318,31 @@ function buildAbstractModelInterface_(dataLayerHelper) {
  * @private
  */
 function processCommand_(command, model) {
-  if (!isString_(command[0])) return;
+  if (!isString_(command[0])) {
+    logError(`Error processing command, no command was run. The first ` +
+        `argument must be of type string, but was of type ` +
+        `${typeof command[0]}. \nYou pushed the command\n ${command}.`,
```

Comment:
This might be the other direction, a bit too many new-lines.. I'd just have the one between the main error (arg 1 was wrong type) and the context (command that caused the problem).

Comment applies to a few other messages.

Reply:
OK I am now more judicial in giving out newlines